### ISSUE TITLE
[v0.3-alpha] MapProperty: Missing handling for key type StrProperty

### ISSUE THREAD

**Issue by SillyBits** [SillyBits/satisfactory-savegame-tool-ng#7]
From error report # 44

```
Message: Reader(New EA.sav|-1): Unknown key type 'StrProperty'
TargetSite: Read, C:\Program Files\Satisfactory Savegame Tool\SavegameHandler.dll
Source: SavegameHandler
StackTrace:
   場所 Savegame.Properties.MapProperty.Read(IReader reader)
   場所 Savegame.Properties.ValueProperty.Read(IReader reader, Property parent)
   場所 Savegame.Properties.PropertyList.Read(IReader reader)
   場所 Savegame.Properties.Entity.Read(IReader reader, Int32 length)
   場所 Savegame.Properties.NamedEntity.Read(IReader reader, Int32 length)
   場所 Savegame.Properties.Actor.ReadEntity(IReader reader)
   場所 Savegame.Savegame._Load(IReader reader)
```

**Comment by SillyBits** [comment]
This is caused by at least mod [Light it up](https://ficsit.app/mod/AT6Zj38KmcKgZ6) if one is using lampgroups, but might also occur with other mods not listed here.

Will add proper handling.

**Comment by SillyBits** [comment]
Fixed in afde76e
